### ISSUE TITLE
LogisticCalibration use _inverse_sigmoid

### ISSUE THREAD

**Issue by JYLFamily** [EFS-OpenSource/calibration-framework#20]
Hi Fabian Küppers
    when use LogisticCalibration() for binary class, why use _inverse_sigmoid(X) rather than X.
    
    version 1.0 netcal    
    `        # if binary, use sigmoid instead of softmax
        if self.num_classes <= 2 or self.independent_probabilities:
            logit = self._inverse_sigmoid(X) 
        else:
            logit = self._inverse_softmax(X)

        # otherwise, use SciPy optimzation. Usually, this is much faster
        if self.num_classes > 2:
            # convert ground truth to one hot if not binary
            y = self._get_one_hot_encoded_labels(y, self.num_classes)

        # if temperature scaling, fit single parameter
        if self.temperature_only:
            theta_0 = np.array(1.0)

        # else fit bias and weights for each class (one parameter on binary)
        else:
            if self._is_binary_classification():
                theta_0 = np.array([0.0, 1.0])
            else:
                theta_0 = np.concatenate((np.zeros(self.num_classes), np.ones(self.num_classes)))

        # perform minimization of squared loss - invoke SciPy optimization suite
        result = optimize.minimize(fun=self._loss_function, x0=theta_0,
                                   args=(logit, y))`

Thanks

**Comment by jwitos** [comment]
@fabiankueppers, just to confirm, all methods in the library expect confidences and _not logits_, correct?

**Comment by fabiankueppers** [comment]
@jwitos exactly
